packages/documentation/docs/for-developers/for-blueprint-developers/HOLD.md

```diff
@@ -0,0 +1,40 @@
+# HOLD
```

Comment:
I feel like this should be just ordinary "Hold". I'm not sure if we should use the all-caps spelling. Maybe @hummelstrand has some opinions?